### PR TITLE
bring in and enforce inclusive language cops

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
https://github.com/ualbertalib/library_applications_development/issues/100

### CODE REVIEW

File: bin/predeploy [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 # This script will be sitting in /home/deploy directory, and is for setting up directories and pulling required files for docker deployment
 # This script should be either rsynced or copied to the UAT server when setting up fresh server.
-# For more information, please see the documentation: https://github.com/ualbertalib/di_internal/blob/master/System-Adminstration/UAT-Environment.md
+# For more information, please see the documentation: https://github.com/ualbertalib/di_internal/blob/master/System-Administration/UAT-Environment.md
```
Comment (jefferya) [inline bin/predeploy:5]:
```suggestion
# For more information, please see the documentation: (https://github.com/ualbertalib/library_applications_development/blob/main/System-Administration/UAT/UAT-Environment.md>
```

File: app/models/jupiter_core/search.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class JupiterCore::Search
 
-  # How dumb is this? Seems pretty dumb, but that's the official recommendation I guess:
+  # How unintelligent is this? Seems pretty unintelligent, but that's the official recommendation I guess:
```
Comment (jefferya) [inline app/models/jupiter_core/search.rb:3]: My personal preference, remove the commentary, something like the following. Feel free to ignore this suggestion. 
```suggestion
  #  official recommendation to return all results in a single page: set to number higher than possible number of results
```

**Review by jefferya — COMMENTED** [review]
Looks good. A few minor suggestions.

File: .rubocop.yml [hunk 3]
```diff
@@ -8,6 +8,10 @@ require:
   - rubocop-performance
   - rubocop-rails
 
+# TODO: update reference once merged.
+inherit_from:
+  - https://raw.githubusercontent.com/ualbertalib/library_applications_development_inclusive_language/cds/initial-commit/inclusive_language_rubocop.yml
```
Comment (jefferya) [inline .rubocop.yml:13]: Leaving a comment at the "TODO"

File: README.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ Some alternatives is you may also use the `localhost` top level domain (e.g: you
 # UAT Environment
 
 The UAT server is accessible on all library staff workstation, and through VPN on any external IP address.  More details regarding access and deployment can be found:
-[Jupiter UAT Setup](https://github.com/ualbertalib/di_internal/blob/master/System-Adminstration/UAT/UAT-Environment.md)
+[Jupiter UAT Setup](https://github.com/ualbertalib/di_internal/blob/master/System-Administration/UAT/UAT-Environment.md)
```
[the remark this answers is not in the document]
Comment (ConnorSheremeta) [reply README.md]: Good catch!

File: app/models/jupiter_core/search.rb [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
   # https://wiki.apache.org/solr/CommonQueryParameters
```
[the remark this answers is not in the document]
Comment (ConnorSheremeta) [reply app/models/jupiter_core/search.rb]: Good catch and suggestion of leveraging IA :+1:

**Comment by ConnorSheremeta** [comment]
Updated with feedback. Once this is merged dependabot should work as expected (https://github.com/ualbertalib/jupiter/pull/3543#issuecomment-2305860262)

**Review by jefferya — APPROVED** [review]
LGTM!

**Pull request merged**